x-pack/solutions/observability/plugins/infra/public/pages/logs/log_entry_categories/sections/top_categories/top_categories_table.tsx

```diff
@@ -94,6 +94,13 @@ export const TopCategoriesTable = styled(
       [expandedCategories, logViewReference, timeRange]
     );
 
+    const tableCaption =
+      topCategories.length === 0
+        ? i18n.translate('xpack.infra.logs.logEntryCategories.noCategoriesMessage', {
```

Comment:
Nit: Since the content specifically refers to the concept table, what do you think about helping the translators by calling this something like `noCategoriesTableCaption`?